bolt/include/bolt/Core/MCPlusBuilder.h

```diff
@@ -740,6 +740,9 @@ class MCPlusBuilder {
     return false;
   }
 
+  /// Return true if the hlt instruction is defind as terminator
+  virtual bool isTerminatorHLT(const MCInst &Inst) const { return false; }
```

Comment:
Lets rename to `isX86HLT()` or alternatively override `isTerminator()` for `X86MCPlusBuilder`.